module/extension/Director/src/director/challenge_priority.cpp

```diff
@@ -58,23 +59,23 @@ namespace module::extension {
         }
 
         // Function to get the priorities of the ancestors of this task
-        auto get_ancestor_priorities = [this](const std::shared_ptr<const BehaviourTask>& task) {
+        auto get_ancestor_priorities = [this](const std::shared_ptr<BehaviourTask>& task) {
             // We are our first ancestor
             std::vector<TaskPriority> ancestors;
```

Comment:
Is this comment implying that this vector should be initialised with a single value?

Reply:
It once upon a time was... and now this comment is wrong